### PR TITLE
Fixed bug in the filters of flyouts

### PR DESCRIPTION
**Pull request by CPAlejandro**
Hello team, we have resolved a bug that did not allow us to use filters in both Miter flyouts and Regulatory Compliance tables.
Closes. #2895.

### CODE REVIEW

**Review by gabiwassan — APPROVED** [review]
LGTM! 
Checked works properly

File: public/components/common/modules/discover/discover.tsx [hunk 1]
```diff
@@ -108,7 +108,7 @@ export const Discover = compose(
   constructor(props) {
     super(props);
     this.KibanaServices = getDataPlugin();
-    this.filterManager = props.shareFilterManager ? this.KibanaServices.query.filterManager : new FilterManager(getUiSettings);
+    this.filterManager = props.shareFilterManager ? this.KibanaServices.query.filterManager : this.KibanaServices.query.filterManager;
```
Comment (Desvelao) [inline public/components/common/modules/discover/discover.tsx:111]: Here, `this.filterManager` always has the same value, don't?

Why not, simplify to:
```
this.filterManager = this.Kibanaservices.query.filerManager
```

**Comment by Desvelao** [comment]
I tested the PR and it is working!

**Review by frankeros — COMMENTED** [review]
I tested it the filters are applied but are using the same Filter manager instance that the kibana discover search bar, it is not expected behavior. 
I proceed to change it.

**Review by gabiwassan — APPROVED** [review]

**Pull request merged**